frontend/src/components/VendorIcon.vue

```diff
@@ -67,7 +67,11 @@ export default {
     },
     iconStyle () {
       const maxIconSize = this.size - 4
-      return `max-height: ${maxIconSize}px; max-width: ${maxIconSize}px; border-radius: 0px;`
+      return {
+        maxHeight: `${maxIconSize}px`,
+        maxWidth: `${maxIconSize}px`,
+        borderRadius: '0px'
```

Comment:
add a class `no-border-radius` to main.scss

Reply:
done, used already defined `rounded-0` class